modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -94,6 +101,62 @@ asmctl api create
 The API that was created is displayed.
 . Optionally, you can verify that an asset is created on Exchange and an API is created on API Manager.
 
+[[create-api-ASM-CLI-ConnAps]]
+== Create an API Using Anypoint Service Mesh With Connected Apps
+
+Use the procedure in this section to automatically create an API using Anypoint Service Mesh with your Connected App credentials. Before you create the API, you must complete the task prerequisite.
+
+*Task Prerequisite*
+
+Before you begin, ensure that:
+
+* You xref:obtain-connected-apps-credentials.adoc[Obtain the client ID and client secret credentials] from Connected Apps
+* The client credentials has the:
+** Exchange Contributor permission to publish the API to Exchange
+** Manage API Configuration to manage the API in API Manager
+
+To create an API using Anypoint Service Mesh with Connected Apps, perform the following steps:
+
+. Using Anypoint Service Mesh, type the following command to create the API:
++
+[source,text,linenums]
+----
+asmctl api create
+--name=_Kubernetes API resource name_ \
+--namespace=_namespace name_ \
+--environmentId=_environment ID_ \
+--groupId=_(Optional) group ID of the asset, for example, organization ID_ \
+--assetId=_asset ID on Exchange_ \
+--version=_asset version of the instance on exchange_ \
+--instanceLabel=_label of the instance on API Manager_ \
+--specTags=_list of comma separated tags to be added to the asset on Exchange_ \
+--instanceTags=_list of comma separated tags to be added to the instance on API Manager_ \
+--clientId=_ID of the Anypoint Platform Connected Apps, using which to create the API instance_ \
+--clientSecret=_client secret of the Anypoint Platform Connected Apps, using which to create the API instance_
+----
++
```

Comment:
Moving the table into a reference section is perhaps a good idea, but I would keep the table with the instructions itself because this information is needed when the users are typing the command. It will be a nuisance to send them elsewhere for each sentence of the command/code. Thanks!